Gemfile

```diff
@@ -3,6 +3,8 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'mysql2'
+gem 'pg'
+gem 'sqlite3'
```

Comment:
You can put development dependencies at gemspec too but you will loose the ability to define bundle groups. The only advantage of having development dependencies is that they will be present on the rubygems pages like this https://rubygems.org/gems/arel.